src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const basePrice = 40;
+  let totalPrice = days * basePrice;
```

Comment:
Prefer `const` over `let` for variables that do not change. Since `totalPrice` is not reassigned but only modified, it can be declared with `const`.